source/stylesheets/refills/_flashes.scss

```diff
@@ -1,40 +1,28 @@
 $base-spacing: 1.5em !default;
-$alert-color: #fff6bf !default;
-$error-color: #fbe3e4 !default;
-$notice-color: #e5edf8 !default;
-$success-color: #e6efc2 !default;
+$flashes: (
+  alert: #fff6bf !default,
```

Comment:
So you can have a `!default` flag here. You just put it at the end, outside of the map:

```scss
$flashes: (
  alert: #fff6bf,
  error: #fbe3e4,
  notice: #e5edf8,
  success: #e6efc2,
) !default;
```

Though, why would we need to? These are encapsulated in a map now, and they would be copied in the source. So just edit the original map directly. If it was wrapped inside a library, the `!default` would be nice.